src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonDateOfBirth.java

```diff
@@ -0,0 +1,44 @@
+package uk.nhs.digital.nhsconnect.nhais.model.edifact;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.nhais.service.TimestampService;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+@Builder
+@Data
+public class PersonDateOfBirth extends Segment {
+    //DTM+329:19911106:102'
+    private final static DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd").withZone(TimestampService.UKZone);
```

Comment:
I've seen this a lot in our code. Can you extract this to TimestampService and change wherever we use that?

Reply:
Where there is only a date we should use LocalDate instead of forcing a nominal time.